test/unit/views/payment-method-view.js

```diff
@@ -204,6 +205,85 @@ describe('PaymentMethodView', function () {
     });
   });
 
+  describe('selecting payment methods', function () {
+    beforeEach(function () {
+      this.client = fake.client();
+      this.model = fake.model();
+      this.sandbox.stub(analytics, 'sendEvent');
+    });
+
+    it('sends an analytic event when a vaulted card is selected', function () {
+      var view = new PaymentMethodView({
+        client: this.client,
+        model: this.model,
+        strings: strings,
+        paymentMethod: {
+          type: 'CreditCard',
+          nonce: 'nonce',
+          vaulted: true,
+          details: {
+            lastFour: 1111
+          }
+        }
+      });
+
+      view._choosePaymentMethod();
+
+      expect(analytics.sendEvent.withArgs(this.client, 'vaulted-card.select')).to.be.calledOnce;
+    });
+
+    it('sends an analytic event when a vaulted paypal payment method is selected', function () {
+      var view = new PaymentMethodView({
+        client: this.client,
+        model: this.model,
+        strings: strings,
+        paymentMethod: {
+          type: 'PayPalAccount',
+          vaulted: true,
+          details: {
+            email: 'test@example.com'
+          }
+        }
+      });
+
+      view._choosePaymentMethod();
+
+      expect(analytics.sendEvent.withArgs(this.client, 'vaulted-paypal.select')).to.be.calledOnce;
+    });
+
+    it.only('does not send an analytic event when no payment is selected', function () {
```

Comment:
Got an `only` here.

Reply:
Good eye. [Fixed](https://github.com/braintree/braintree-web-drop-in/pull/508/commits/09a56806e2391be2881354de6afa12f3d7037219)!

Reply:
Squash before merge?

Reply:
Sure